core/src/main/java/com/microsoft/applicationinsights/common/CommonUtils.java

```diff
@@ -41,6 +44,8 @@ public static String getHostName(){
         }
         catch (UnknownHostException ex) {
             // optional parameter. do nothing if unresolvable
+            InternalLogger.INSTANCE.trace("Unresolvable host error");
+            InternalLogger.INSTANCE.trace("Stack trace generated is %s", ExceptionUtils.getStackTrace(ex));
```

Comment:
@dhaval24 There's no reason to call the logger twice on the same level. Feel free to use `%n` to insert a newline between the messages if you want them on separate lines.